### PR TITLE
Create 08_Tool_Sets.txt

### PR DESCRIPTION
**Pull request by Turtlelord26**
#932 

Proposed templating for tool set items with examples, in a new, dedicated file. Having tool sets exist makes it easier to add them to profession backgrounds. Also prose is appearing.

Medkit and repair kit should probably move to this page from 07_items, along with a couple other things.

### CODE REVIEW

**Review by rosebrallier — COMMENTED** [review]
This is a fantastic start!
I'm thinking that when characters attempt a skill check without the necessary tools, they roll at disadvantage, and in some cases may have to roll jury-rigging at disadvantage instead of the more pertinent skill.

**Comment by rosebrallier** [comment]
I'm going to have @Reveraine pile on this and fill out some of the prose. @dfuchs2 you can add to this as well, as your time allows. Follow the completed sets as a template.

**Comment by Turtlelord26** [comment]
This may be approaching basic functionality @rosebrallier 

It would be helpful to the current iteration of the Assisted Skills mechanic (requires tools) to separate out some skills, such as lockpicking from sleight of hand, so that I can define what can be used without tools from the skills side, but I'm not sure if we want to increase skill granularity like that. I am also hesitant to try and define that from the toolkit end of the mechanic, as it would be harder for players to look up. Maybe it's less of a problem than I think and I should just leave it entirely to the GMs whether players can do [A Thing] with a particular skill and [lack of] tool.

**Comment by Turtlelord26** [comment]
Explorers prep would love to have fewer branches to trip people with.

File: Items/08_Tool_Sets.txt [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+Weight: 
+
+Electrician's Kit ($350):
+	Voltage testers; sensors; insulated wire strippers, ratchets, 
+screwdrivers, and pliers; reels of wire; soldering supplies; breadboards; 
+resistors; capacitors; inductors; rechargeable temporary power supplies; 
+and a miniaturized 3D circuit printer allow the professional electrician 
+to maintain, repair, and construct all manner of electronic gadgetry.
+
+Relevant Skill(s): Repair, Jury-rig
```
Comment (rosebrallier) [inline Items/08_Tool_Sets.txt]: *Jury-rigging

**Review by rosebrallier — COMMENTED** [review]
We've made some darn good progress here. I've even acquired some inspiration for additional prose.
In other news, let's defer adding weight info to any of these yet. If nothing has a weight, it'll be harder to miss one when we develop the relevant systems.

**Comment by rosebrallier** [comment]
The remaining required prose is not particularly within my areas of interest, experience or expertise.
@bleehu we need your input on a few of these, and well as your review of the prices re: game economy.
@Reveraine See what else you can chuck into the prose pot?
@Turtlelord26 I approve of what you've done with Assisted skills. Gives a framework for the GM and players to play off of without getting too...nitpicky.

File: CharacterCreation/06_Skills.txt [hunk 2]
```diff
@@ -65,6 +65,11 @@ this is not an exhaustive list of skills, merely the skills called for most
 often. Talk to your GM if you would like your character to learn a more specific
 case of one of these skills or a skill not listed below.
 
+	Assisted skills, which usually require an appropriate tool set as 
+described in the basic rules, are marked with an [A]. Keep in mind that, regardless 
+of [A]'s, having an appropriate set of tools for the task at hand will help with 
+almost any skill, though.
```
Comment (rosebrallier) [inline CharacterCreation/06_Skills.txt]: We may want to make an explicit exception here for Reload checks. They're Weapons skill checks, but shouldn't require an entire skill check to perform.

File: Items/08_Tool_Sets.txt [hunk 3]
```diff
@@ -116,10 +116,12 @@ Noise Level: Low
...
 
 Fine Art Kit ($150):
-	[ph]
+	Tools for either 2D or 3D physical fine art disciplines such as paints,
+brushes and canvasses, pencils, markers and paper, or sculpting and carving tools
+and a project's supply of the artist's medium of choice.
 
 Relevant Skill(s): Performance
-Context: 2D physical Art (eg. painting)
+Context: Physical fine art (eg. painting, sculpting)
```
Comment (Turtlelord26) [inline Items/08_Tool_Sets.txt]: See Sculptor's Kit, below

**Comment by Turtlelord26** [comment]
In-scope: Refill kits for the toolsets flavored as containing consumables?

File: Items/08_Tool_Sets.txt (hunk 3 above)
Comment (rosebrallier) [reply Items/08_Tool_Sets.txt]: Herp derp. Not sure why I didn't see that when I was reading the musical instrument entry.

**Comment by rosebrallier** [comment]
> In-scope: Refill kits for the toolsets flavored as containing consumables?

Sure; we could duplicate the ammo pricing philosophy.

File: 01_01_Basic_Rules.txt [hunk 4]
```diff
@@ -563,15 +563,32 @@ have invested in learning the skill; however, your GM may decide otherwise, appl
...
-	When leveling up you may gain up to 6 points in any one skill.
-	
-	Characters are considered to be Proficient with certain skills, based on
-their class and background. When a character is Proficient in a skill, each skill 
-point they spend in that skill upon leveling up increases that skill by two 
-instead of one.
-
 	Defenders win ties for opposing rolls.
 
+-- Assisted Skills and Tool Sets--
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: Callling a skill "assisted" brings to mind two characters helping on the same skill check, as opposed to a character using a tool. Maybe something like "instrumented skill check"? Though I admit that doesn't have a nice ring to it.

File: 01_01_Basic_Rules.txt [hunk 5]
```diff
@@ -563,15 +563,32 @@ have invested in learning the skill; however, your GM may decide otherwise, appl
...
+
+	When a skill is marked Assisted, it pertains to many situations that cannot 
+in general be done well without the correct tools. Assisted skill rolls made 
+without the correct tools are rolled with Disadvantage, and may even be impossible. 
+At the GM's discretion, it may be possible to improvise a tool for a task, 
+probably requiring a jury-rigging roll, but this may impose complications.
+
+	Other skills suffer no penalties to use without a tool set, but an 
+appropriate one to the task at hand may still provide a bonus. Unless otherwise 
+specified, augmenting a roll with an appropriate tool set provides +10 to the roll.
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: Good default. I like this.

File: 01_01_Basic_Rules.txt [hunk 6]
```diff
@@ -625,4 +642,4 @@ out a receiver, you may carry (at max) one additional receiver in order to swap
 kits on the go. You must pass 2 Weapons use / knowledge [Weapons] checks in 
 order to successfully convert your weapon on the go. DC 60. Failure on one check 
 instantly jams the weapon on first shot. Failure on both checks requires an 
-additional check.
\ No newline at end of file
+additional check.
```
Comment (bleehu) [inline 01_01_Basic_Rules.txt]: EOF changed from windows/linux standard. Not sure why. Shouldn't break anything. Don't really care.

File: Items/08_Tool_Sets.txt [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+
+Blade Maintenance and Care Kit ($100):
+	A good whetstone, polish, oil and cleaning supplies are worth
+their weight in gold. These tools keep blades sharp, rust-free and
+ready for action.
+
+Relevant Skill(s): Weapons - Melee, Weapons - Thrown
+Context: Bladed Weapons
+Noise Level: Low
+Weight:
```
Comment (bleehu) [inline Items/08_Tool_Sets.txt:13]: Let's remove unknown weights and add them back in when we write encumbrance rules?

File: Items/08_Tool_Sets.txt [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+	A good whetstone, polish, oil and cleaning supplies are worth
+their weight in gold. These tools keep blades sharp, rust-free and
+ready for action.
+
+Relevant Skill(s): Weapons - Melee, Weapons - Thrown
+Context: Bladed Weapons
+Noise Level: Low
+Weight: 
+
+Digital Artist's Program and Tablet Package ($250):
```
Comment (bleehu) [inline Items/08_Tool_Sets.txt:15]: This is brilliant! Forge documents? Oh ya!

Comment (bleehu) [reply Items/08_Tool_Sets.txt:15]: Let's drop the price of this just a bit; I expect it to come up fairly infrequently.

File: Items/08_Tool_Sets.txt [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+TIR Digital Painting, which can be used for drawing, painting, and 
+editing photos and documents. Comes with a protective case and 
+rechargeable 8-hour battery.
+
+Relevant Skill(s): Performance, Deception
+Context: Digital Art, Imagery
+Noise Level: Low
+Weight:
+
+Destructive Entry Kit ($200):
```
Comment (bleehu) [inline Items/08_Tool_Sets.txt:26]: Should this replace the similar tools in the tool section of items?

File: Items/08_Tool_Sets.txt [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+to maintain, repair, and construct all manner of electronic gadgetry.
+
+Relevant Skill(s): Repair, Jury-rigging
+Context: Electronics, Electrical Components, Circuitboards. Repair, 
+	construction, repurposing.
+Noise Level: Low
+Weight:
+
+Electronic Lock Spoofing Kit ($250):
+	[ph]
```
Comment (bleehu) [inline Items/08_Tool_Sets.txt]: Blocking from merging until place holders are filled.

File: Items/08_Tool_Sets.txt [hunk 11]
```diff
@@ -0,0 +1,215 @@
...
+
+General Research Analysis and Cross-referencing AI ($350):
+	A portable hand terminal containing a massive solid-state data drive 
+and a smaller drive with a non-sentient AI specialized in data retrieval. 
+The AI can help investigate inquiries by intelligently loading up reputable 
+documents and providing summaries. Performance greatly improved by access to 
+academic or library databanks. Takes time to achieve best results. Comes with 
+a protective case and rechargeable 8-hour battery.
+
+Relevant Skill(s): Knowledge - All, Investigation
```
Comment (bleehu) [inline Items/08_Tool_Sets.txt:100]: I'm really torn on this item. On the one hand, it seems OP. On the other hand, it's got a price tag to match and wikipedia is a real thing.

File: Items/08_Tool_Sets.txt [hunk 12]
```diff
@@ -0,0 +1,215 @@
...
+Context: Research
+Noise Level: Low
+Weight: 
+
+Hostile Environment General Purpose Survival Package ($600):
+	[ph][one kit does for a party]
```
Comment (bleehu) [inline Items/08_Tool_Sets.txt]: Too general; let's have different kits for each hostile environment.

File: Items/08_Tool_Sets.txt [hunk 13]
```diff
@@ -0,0 +1,215 @@
...
+
+Relevant Skill(s): Survival
+Context: Mitigating thermally extreme, low-gravity, and/or nonstandard atmospheres
+Noise Level: Low
+Weight: 
+
+Field-expedient Trauma Kit ($400):
```
Comment (bleehu) [inline Items/08_Tool_Sets.txt]: Request change: Let's fold this item and the Doctor's field applicator together into one item. I don't want the Doc to feel like they need to draw two separate tools to get their best healing done.

Comment (bleehu) [reply Items/08_Tool_Sets.txt]: I suggest that medicine and repair stay separate. Blending them here is the first instance of the anti-pattern of combining them.

**Review by bleehu — CHANGES_REQUESTED** [review]
Pretty cool new system! Gonna need at least one more iteration before it's ready for dev.

File: Items/08_Tool_Sets.txt (hunk 9 above)
Comment (Turtlelord26) [reply Items/08_Tool_Sets.txt:26]: My endgame is to absorb that entire section into this document

File: Items/08_Tool_Sets.txt (hunk 11 above)
Comment (Turtlelord26) [reply Items/08_Tool_Sets.txt:100]: It's also entirely controlled by the GM and basically useless in combat.

**Review by bleehu — APPROVED** [review]

**Pull request closed without merging**